src/apis/UserApi.ts

```diff
@@ -94,4 +94,22 @@ export interface UserApi {
     eventFired: (user: User) => Promise<void>,
     options?: { keepAlive?: number; lifetime?: number; connectionMaxRetry?: number; connectionRetryIntervalMs?: number }
   ): Promise<Connection>
+
+  /**
+   * Add autoDelegations values to the user.
+   * @param type Type of AutoDelegation to add
+   * @param user User to update
+   * @param to Array of DataOwnerId to add
+   * @return Updated user
+   */
+  addAutoDelegationsTo(type: DelegationTag, user: User, to: string[]): Promise<User>
```

Comment:
Rename it to `shareAllFutureDataWith(sharedDataType: SharedDataType, dataOwnerIds: string[])`